avalanche/training/plugins/evaluation.py

```diff
@@ -44,20 +43,16 @@ def __init__(
         :param collect_all: if True, collect in a separate dictionary all
             metric curves values. This dictionary is accessible with
             `get_all_metrics` method.
-        :param benchmark: DEPRECATED. Ignored argument.
         :param strict_checks: if True, checks that the full evaluation streams
             is used when calling `eval`. An error will be raised otherwise.
             When False only warnings will be raised.
-        :param suppress_warnings: if True, warnings and errors will never be
-            raised from the plugin.
-            If False, warnings and errors will be raised following
-            `benchmark` and `strict_checks` behavior.
         """
         super().__init__()
         self.collect_all = collect_all
-        self.benchmark = benchmark
         self.strict_checks = strict_checks
-        self.suppress_warnings = suppress_warnings
+
+        self.benchmark = benchmark
```

Comment:
Why the `EvaluationPlugin` does not check benchmark anymore? I know it will cause regressions but in this case I think it would be cleaner to remove it from the code.

Reply:
you mean removing `self.benchmark`? it's used like before. The only difference is that `self.benchmark` is initialized automatically using the experience. This also explains why we still have `strict_checks`.

Reply:
I meant removing `benchmark` from the `__init__` signature, since I noticed it has been removed from the APIdoc. If the user can still provide it as input it's better to add it again in the APIdoc.